recipes/r-betareg/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '3.1-0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-betareg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: betareg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/betareg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/betareg/betareg_{{ version }}.tar.gz
+  sha256: c7ddf21181e14fcceb2101ae0b88d6753bbcc97fbd81b96137280821bfa64517
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-formula
+    - r-flexmix
+    - r-lmtest
+    - r-modeltools
+    - r-sandwich
+
+  run:
+    - r-base
+    - r-formula
+    - r-flexmix
+    - r-lmtest
+    - r-modeltools
+    - r-sandwich
+
+test:
+  commands:
+    - $R -e "library('betareg')"  # [not win]
+    - "\"%R%\" -e \"library('betareg')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=betareg
+  license: GPL-2 | GPL-3
+  summary: Beta regression for modeling beta-distributed dependent variables, e.g., rates and
+    proportions. In addition to maximum likelihood regression (for both mean and precision
+    of a beta-distributed response), bias-corrected and bias-reduced estimation as well
+    as finite mixture models and recursive partitioning for beta regressions are provided.
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
```

Comment:
You can use `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'` or `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2' instead of copying the license into the recipe. Both are okay and it's upto you.